pkg/nuctl/command/import.go

```diff
@@ -114,17 +116,37 @@ func (i *importCommandeer) importFunction(ctx context.Context, functionConfig *f
 func (i *importCommandeer) importFunctions(ctx context.Context,
 	functionConfigs map[string]*functionconfig.Config,
 	project *platform.ProjectConfig) error {
-	errGroup, _ := errgroup.WithContext(ctx, i.rootCommandeer.loggerInstance)
-
+	importFailed := atomic.Bool{}
 	i.rootCommandeer.loggerInstance.DebugWithCtx(ctx, "Importing functions", "functions", functionConfigs)
+	wg := sync.WaitGroup{}
 	for _, functionConfig := range functionConfigs {
-		functionConfig := functionConfig // https://golang.org/doc/faq#closures_and_goroutines
-		errGroup.Go("Import function", func() error {
-			return i.importFunction(ctx, functionConfig, project)
-		})
+		wg.Add(1)
+		go func(function *functionconfig.Config) {
+			i.rootCommandeer.loggerInstance.DebugWithCtx(ctx, "Importing function",
+				"function", function.Meta.Name,
+				"project", project.Meta.Name)
+			if err := i.importFunction(ctx, function, project); err != nil {
+				if !importFailed.Load() {
+					importFailed.Store(true)
+				}
```

Comment:
perhaps use sync map `map[project.name][function.name] = false` so that we can better log it to the user

Reply:
@liranbg we will definitely need to add this in future PR (with report file functionality), but within this PR it will just be a log duplication

Reply:
@rokatyy Add this as a TODO then.